packages/eas-cli/src/credentials/ios/utils/provisioningProfile.ts

```diff
@@ -26,6 +26,11 @@ export function isAdHocProfile(dataBase64: string): boolean {
   return Array.isArray(provisionedDevices);
 }
 
+export function isEnterpriseUniversalProfile(dataBase64: string): boolean {
+  const profilePlist = parse(dataBase64);
+  return !!profilePlist['ProvisionsAllDevices'];
```

Comment:
Compare with undefined/null. Remember the enum issue from a few days ago? XD

Reply:
That issue was caused by the fact that falsy value was used as sth that represented some state. Here we are checking whether the value exists and regardless of whether it is null/undefined or any other falsy value it should work the same way. Comparing explicitly against undefined is in my opinion risky because it's easy to introduce bugs where null or undefined are mixed and not properly reflected by a type system.